src/blog/lagom-1.4.6.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Lagom 1.4.6 released!
+date: 2018-05-31
+author_github: ignasi35
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.4.6
+---
+
+The Lagom team has released Lagom 1.4.6.
+
+This release provides several bug fixes and updates.
+
+## Highlights:
+
+- Updates Akka to [2.5.13](https://akka.io/blog/news/INVALID_URL)
```

Comment:
Akka will be publishing their release notes shortly. Then we can fix the link and publish.